backend/telemetry/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryLocalStorage.java

```diff
@@ -419,4 +436,22 @@ public int getExportedConnectedModeCount() {
     return exportedConnectedModeCount;
   }
 
+  public void addBoundSQSProjectKey(String projectKey) {
+    markSonarLintAsUsedToday();
+    boundSQSProjectKeys.add(projectKey);
+  }
+
+  public Set<String> getBoundSQSProjectKeys() {
+    return boundSQSProjectKeys;
+  }
+
+  public void addBoundSQCProjectKey(String projectKey) {
+    markSonarLintAsUsedToday();
+    boundSQCProjectKeys.add(projectKey);
+  }
+
+  public Set<String> getBoundSQCProjectKeys() {
```

Comment:
Nitpick: maybe just a personal preference, I prefer not to use abbreviations in code as much as possible. Here SQS and SQC are very close, so it's easy to make a mistake and hard to spot it. I think we could afford a few extra characters to make things clearer